docs/exercise008-bonus-activity.md

```diff
@@ -0,0 +1,83 @@
+# Exercises 008 - Optional Bonus Activity!
+
+Congratulations for finishing these JavaScript challenges!
+
+You have covered so many different JavaScript concepts and skills in a very short space of time, including:
+- Problem Solving
+- For Loops
+- ES6 methods
+- Conditionals
+- Data Types
+- Objects
+and more!
+
+In these exercise we would like you to take a look at two new features of ES6, the Spread Operator and Optional Chaining.
```

Comment:
"exercises"

Reply:
Ahhh brill, I meant to ask/confirm this and completely forgot! I'd googled it but couldn't find when exactly it came out. I'll update :)